### PR TITLE
CHE-283 Display machine status next to the dev-machine

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
**_1 Upvote**_ Signed-off-by: Vitaliy Guliy vguliy@codenvy.com

### CODE REVIEW

File: core/platform-api-client-gwt/che-core-client-gwt-workspace/src/main/java/org/eclipse/che/api/workspace/gwt/client/event/WorkspaceStoppedHandler.java [hunk 1]
```diff
@@ -8,24 +8,23 @@
...
  *
- * @author Dmitry Shnurenko
+ * @author Vitaliy Guliy
  */
-public interface StopWorkspaceHandler extends EventHandler {
+public interface WorkspaceStoppedHandler extends EventHandler {
 
     /**
-     * Performs some actions when workspace is stopped.
+     * Perforrm actions when workspace is stopped.
```
Comment (azatsarynnyy) [inline core/platform-api-client-gwt/che-core-client-gwt-workspace/src/main/java/org/eclipse/che/api/workspace/gwt/client/event/WorkspaceStoppedHandler.java]: typo

File: core/platform-api-client-gwt/che-core-client-gwt-workspace/src/main/java/org/eclipse/che/api/workspace/gwt/client/event/WorkspaceStoppedHandler.java [hunk 2]
```diff
@@ -8,24 +8,23 @@
...
-package org.eclipse.che.ide.workspace.start;
+package org.eclipse.che.api.workspace.gwt.client.event;
 
 import com.google.gwt.event.shared.EventHandler;
 
-import org.eclipse.che.api.workspace.shared.dto.UsersWorkspaceDto;
-
 /**
- * Provides methods which allows do some actions when workspace is stopped
+ * Is fired when workspace has been stopped.
```
Comment (azatsarynnyy) [inline core/platform-api-client-gwt/che-core-client-gwt-workspace/src/main/java/org/eclipse/che/api/workspace/gwt/client/event/WorkspaceStoppedHandler.java:16]: is it a question? ))

**Comment by ashumilova** [comment]
+1 after typo fixes

File: core/platform-api-client-gwt/che-core-client-gwt-workspace/src/main/java/org/eclipse/che/api/workspace/gwt/client/event/WorkspaceStoppedHandler.java (hunk 1 above)
Comment (vitaliy-guliy) [inline core/platform-api-client-gwt/che-core-client-gwt-workspace/src/main/java/org/eclipse/che/api/workspace/gwt/client/event/WorkspaceStoppedHandler.java]: Thanks

**Comment by vparfonov** [comment]
ok

**Pull request merged**